README.md

```diff
@@ -426,6 +427,28 @@ EXAMPLE
   $ sfdx sfpowerkit:project:manifest:merge -p project1/path/to/package.xml,project2/path/to/package.xml -d result/package.xml
 ```
 
+### `sfpowerkit:project:datamodel:diff`
+
+Provides an audit history of the metadata change between two commit ID's
```

Comment:
```suggestion
Provides an audit history of the metadata change between two commit ID's
```
Provides an audit history of the metadata change between two commit ID's for data model ( CustomFields, RecordTypes, BusinessProcess)